keep-ui/next.config.js

```diff
@@ -41,8 +41,8 @@ const nextConfig = {
       : [
           {
             source: "/",
-            destination: "/providers",
-            permanent: true,
+            destination: "/incidents",
+            permanent: process.env.ENV === "production",
```

Comment:
why? 👀

Reply:
It's breaking local dev env if you run multiple things at http://localhost:3000 one after another. Once you touch keep, it will always redirect to http://localhost:3000/providers :)